core/config/config.go

```diff
@@ -2,6 +2,10 @@
 // Chain Core.
 package config
 
+// The following go:generate command is included in order to generate code for
```

Comment:
mild suggestion:

s/The following go:generate command is included in order to//